docs/content/usage/configuration/credentials.md

```diff
@@ -5,6 +5,13 @@ draft: false
 description: "Configure credentials that the Infection Monkey will use for propagation."
 ---
 
-On this screen you can feed the Infection Monkey "stolen" credentials from your network, simulating an attacker with inside knowledge.
+On this screen, you can configure credentials for Infection Monkey to use in the propagation
+stage. These can be anything from default or weak passwords, to "stolen" credentials from your
+network, simulating an attacker with inside knowledge.
+
+All possible credential combinations will be tried for propagation. Identity-secret pairs added
+in the same row will be attempted before any other credential combinations.
```

Comment:
```suggestion
All possible credential combinations will be tried for propagation. Credentials on the same row are treated as if they belong to the same user and are tried first.
```

Reply:
Defined identities and secrets. I like it the way it is now. We can talk about it if you feel strongly against it.